### ISSUE TITLE
Potential redesign of crafting mechanics advised

### ISSUE THREAD

**Issue by EikoBiko** [ChiselsAndBits/Chisels-and-Bits#1031]
Right now a couple crafting mechanics seem arbitrarily limiting. I will explain:

Imagine you are chiseling a generic 2x2 table. 
![image](https://user-images.githubusercontent.com/53350053/204077957-a3efbad8-5b2d-47f7-bb55-e153cd088319.png)
To create this, it would be sensible to make 1 corner of the table, and rotate it 3 times to create the other 3 corners. But in order to use the same pattern multiple times, you must currently "seal" it. However, you also can't rotate a sealed pattern. So if you want rotated versions of that pattern, you must:
1.) Use the modification table to create 4 separate (but identical) patterns at different rotations. (This seems unnecessarily tedious.)
**_OR_**
2.) Place the block 4 times and pick them up, then turn them individually. (!)

So, since you can place the design and _**then**_ turn it, does the modification table have a reason to exist? Then, if the modification table is rendered pointless because of free pattern rotation, isn't the unsealed design just an inferior version of the sealed design?

In short, the two major design questions for the current iteration:
1.) What is the purpose for requiring a workbench to rotate a specific type of pattern when players can already do it without the workbench?
2.) Considering that sealed pattern rotation is possible, how do single-use patterns improve the player's experience? Consider that the resource management portion is already accomplished by the bits themselves -- doesn't this make single-use patterns entirely redundant?
